package-examples/ghost/ghost/Kptfile

```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: ghost
```

Comment:
ghost pkg inside ghost confused me a bit :) Will think of an alternative as I go further in the PR.

Reply:
chose ghost-app. Updated.

Reply:
I've been thinking we need to provide the full package path of any subpackage as a string, such as toppkg-middlepkg-leafpkg. Maybe that's what should populate the subpackage package-context. Or maybe we should add an additional field to it containing the full name.

For example, if the top-level package name were "ghost" and this subpackage's name were "app", then then full name would be "ghost-app".